.github/workflows/build.yml

```
@@ -8,7 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java_version: [11, 17, 18, 19-ea]
+        java_version: [11, 17, 18]
```

```suggestion
        java_version: [11, 17, 18]
        experimental: [false]
```
To set the flag for other builds